packages/core/test-helpers/core-test-helpers-model-versions/src/types.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Client } from '@elastic/elasticsearch';
+import type { SavedObjectsType } from '@kbn/core-saved-objects-server';
+import type {
+  ISavedObjectsRepository,
+  SavedObjectsBulkCreateObject,
+} from '@kbn/core-saved-objects-api-server';
+
+/**
+ * A testbed that can be used for model version integration tests.
+ *
+ * @public
+ */
+export interface ModelVersionTestBed {
```

Comment:
Naming is hard... We have a `testbed` returning a `testkit`... 

Probably not that important, but still, if anyone has a proposal on a better naming I'm all ears.

Reply:
The only alternative I can think of right now is `testSheet` 😆